docs/man/docker.1.md

```diff
@@ -292,79 +292,192 @@ inside it)
 
 # STORAGE DRIVER OPTIONS
 
-Options to storage backend can be specified with **--storage-opt** flags. The
-only backend which currently takes options is *devicemapper*. Therefore use these
-flags with **-s=**devicemapper.
+Docker makes use of storage backends (or "graphdriver" in Docker
+internals) to create writable containers from images.  Many of these
+backends make use of operating system level technologies, and support
+configuration.
+
+Options to the storage backend can be specified with **--storage-opt**
+flags. The only backend which currently takes options is
+*devicemapper*. Therefore use these flags with **-s=**devicemapper.
+
+Specifically for devicemapper, the default is a "loopback" model which
+requires no pre-configuration, but is extremely inefficient.  Do not
+use it in production.
+
+In order to make the best use of Docker with the devicemapper backend,
+you must have a recent version of lvm2.  Use `lvm` to create a thin
+pool.  Then, use `--storage-opt dm.thinpooldev` to tell the Docker
+engine to use that pool for allocating images and container snapshots.
 
 Here is the list of *devicemapper* options:
 
+#### dm.thinpooldev
+
+Specifies a custom blockdevice to use for the thin pool.
+
+If using a block device for device mapper storage, ideally lvm2 would
+be used to create/manage the thin-pool volume that is then handed to
```

Comment:
This sentence is awkward.  Split into two sentences to reduce complexity and increase clarity. Maybe:

If using a block device for device mapper storage, it is best to use `lvm2` to create/manage the thin-pool volume. This volume is then handed to
Docker to exclusively create/manage the thin and thin snapshot volumes
needed for its containers.